robot_controllers/src/follow_joint_trajectory.cpp

```diff
@@ -420,7 +420,8 @@ void FollowJointTrajectoryController::executeCb(const control_msgs::FollowJointT
       executable_trajectory = new_trajectory;
 
       // if this hasn't started yet, need to insert current position
```

Comment:
```suggestion
      // if this hasn't started yet or if the header stamp is in the future, need to insert current position
```

Reply:
I just updated the comment using GitHub, but GitHub might have messed with the whitespace?

Reply:
Thanks, I updated!